### PR TITLE
doc: add utilisation to readme

### PR DESCRIPTION
**Pull request by Zahrun**
This is my current understanding of how cryptotithe works. Please correct me if necessary.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,3 +15,16 @@ Demo - ipfs(infura) - check latest build
   2. Run `yarn`
   3. Run `yarn run build`
   4. Run `yarn run start`
+
+## Utilisation
+Cryptotithe welcomes the user with the choice of loading a existing data file from disk or to create a new empty data file to disk.
+  1. The "Home" tab contains the portfolio with automatically calculated current holdings in all currencies.
+  2. The "Trades" tab contains an history of all trades, which can be filtered by exchange and by currency in timeline format or in table format.
```
Comment (starsoccer) [inline README.md:22]: ` The "Trades" tab contains an history of all trades` -> ` The "Trades" tab contains a history of all trades`

Comment (Zahrun) [reply README.md]: Is the rest of what I wrote correct?

Comment (starsoccer) [reply README.md]: Yes the rest of what you wrote is correct. If your good with me merging this I will

Comment (Zahrun) [reply README.md]: Thank you for clarifying. You can merge it.

**Pull request merged**